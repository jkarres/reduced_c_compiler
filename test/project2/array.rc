
int[10] ia;

function : void main() {

  float[10] fa;

  int i;

  for (i = 0; i < 10; ++i) {

      fa[i] = ia[i] = 10-i;

  }

  for (i = 0; i < 10; ++i) {

    cout << ia[i] << " " << fa[i] << endl;

  }

  bool[10] ba;

  bool * pb = &ba[0];


  for (i = 0; i < 10; ++i) {

    pb[i] = (ia[i] <= 5);

  }

  for (i = 0; i < 10; ++i) {

    cout << ba[i] << endl;

  }

}
