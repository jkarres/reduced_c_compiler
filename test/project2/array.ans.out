10 10.00
9 9.00
8 8.00
7 7.00
6 6.00
5 5.00
4 4.00
3 3.00
2 2.00
1 1.00
false
false
false
false
false
true
true
true
true
true
